canonical-websites/www.ubuntu.com
# Rewrite Ubuntu navigation scss for Vanilla 1.7.0

## Done

- Rewrote and greatly reduced the amount of bespoke code in the current `.p-navigation` pattern
- Removed bespoke search box in favour of Vanilla `.p-search-box`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check that the nav looks nice and Vanilla-y
- Test primary nav (the big orange one), secondary nav (e.g /download) and tertiary nav (e.g. /download/server) in all viewport sizes to make sure it doesn't break

## Issue / Card

Fixes vanilla-framework/vanilla-framework#1715
Fixes vanilla-framework/vanilla-framework#1723
